sysconfig/sysconfig.go

```diff
@@ -111,7 +111,7 @@ func ApplyFilesystemOnlyDefaults(model *asserts.Model, rootDir string, defaults
 // It is okay to use both from install mode for run mode, as well as from the
 // initramfs for recover mode.
 func ConfigureTargetSystem(model *asserts.Model, opts *Options) error {
```

Comment:
I didn't consider it when I made the change to pass model, but this function is meant so far to be called only on UC20 and the way now configureCloudInit looks at grade makes it even more evident. I think we should probably clarify this in the doc comment, and also add a check here about the model and return internal error if it's not a UC20 model.

Reply:
good point, I honestly had to check myself when implementing the new logic to check if it needed to fail gracefully for non-uc20 usage, so having this check is definitely the right thing to do. thanks for the suggestion